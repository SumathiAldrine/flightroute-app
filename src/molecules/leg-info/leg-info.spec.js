import React from 'react';
import renderer from 'react-test-renderer';
import LegInfo from './index';

test('Render Leg Information', () => {
  const tree = renderer.create(<LegInfo />).toJSON();
  
  expect(tree).toMatchSnapshot();
});