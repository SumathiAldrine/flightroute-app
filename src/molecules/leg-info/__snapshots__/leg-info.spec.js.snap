// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render  1`] = `
<div
  className="leg-information__leg-info"
>
  <div
    className="leg-information__leg-info-name"
    tabIndex="0"
  />
  <div
    className="leg-information__leg-info-time"
    tabIndex="0"
  />
  <div
    tabIndex="0"
  />
</div>
`;

exports[`Render Leg Information 1`] = `
<div
  className="leg-information__leg-info"
>
  <div
    className="leg-information__leg-info-name"
    tabIndex="0"
  />
  <div
    className="leg-information__leg-info-time"
    tabIndex="0"
  />
  <div
    tabIndex="0"
  />
</div>
`;

exports[`renders learn react link 1`] = `
<div
  className="leg-information__leg-info"
>
  <div
    className="leg-information__leg-info-name"
    tabIndex="0"
  />
  <div
    className="leg-information__leg-info-time"
    tabIndex="0"
  />
  <div
    tabIndex="0"
  />
</div>
`;
